apps/emqx_authn/src/simple_authn/emqx_authn_jwt.erl

```diff
@@ -432,7 +434,7 @@ verify_claims(Claims, VerifyClaims0) ->
     Now = os:system_time(seconds),
     VerifyClaims =
         [
-            {<<"exp">>, required, fun(ExpireTime) ->
+            {<<"exp">>, optional, fun(ExpireTime) ->
```

Comment:
Nit: the whole code related to `optional/required` was added to make `exp` required, maybe we may remove it, since there are no `required` claims anymore.